source/type/type.h

```diff
@@ -614,12 +614,14 @@ struct Line : public Header, Nameable, HasMessages, Codes{
     MultiLineString shape;
     boost::optional<boost::posix_time::time_duration> opening_time, closing_time;
 
+    std::map<std::string,std::string> properties;
```

Comment:
No I agree with you, since it is needed only for lines for the moment, it's ok to implement it only for that.